glass/src/main/java/io/p13i/glassnotes/activities/EditActivity.java

```diff
@@ -197,12 +253,35 @@ public void rejected(Throwable t) {
                 // ctrl-x is save and finish activity
                 saveAndFinish();
                 return true;
+            } else if (keyCode == KeyEvent.KEYCODE_R) {
+                if (mTopScoredDocument != null) {
+                    Note note = getNoteWithFileName(mTopScoredDocument.getDocument().getContext().getSubject());
```

Comment:
Separate out the `getSubject` line into a new line
```suggestion
                    String noteSubject = mTopScoredDocument.getDocument().getContext().getSubject();
                    Note note = getNoteWithFileName(noteSubject);
```